BREAKING.md

```diff
@@ -32,7 +32,7 @@ ICodeLoader interface was deprecated a while ago and will be removed in the next
 - [Scope is no longer an IFluidObject](#scope-is-no-longer-an-IFluidObject)
 - [IFluidHandle and requestFluidObject generic's default no longer includes IFluidObject](#IFluidHandle-and-requestFluidObject-generics-default-no-longer-includes-IFluidObject)
 - [LazyLoadedDataObjectFactory.create no longer returns an IFluidObject](#LazyLoadedDataObjectFactory.create-no-longer-returns-an-IFluidObject)
-
+- [Remove write method from IDocumentStorageService](#Remove-Write-Method-from-IDocumentStorageService)
```

Comment:
Following the same format for the 0.59 section, can you create and add this to a new 1.0 section above?  (0.59 release branch already got cut and next is bumping to 1.0 imminently)